python/lsst/meas/base/plugins.py

```diff
@@ -78,6 +78,16 @@
 wrapSimpleAlgorithm(bl.BlendednessAlgorithm, Control=bl.BlendednessControl,
                     TransformClass=bl.BaseTransform, executionOrder=BasePlugin.SHAPE_ORDER)
 
+wrapTransform(bl.PsfFluxTransform)
+wrapTransform(bl.PeakLikelihoodFluxTransform)
+wrapTransform(bl.GaussianFluxTransform)
+wrapTransform(bl.GaussianCentroidTransform)
+wrapTransform(bl.NaiveCentroidTransform)
+wrapTransform(bl.SdssCentroidTransform)
+wrapTransform(bl.SdssShapeTransform)
+wrapTransform(bl.ScaledApertureFluxTransform)
+wrapTransform(bl.ApertureFluxTransform)
```

Comment:
Never mind. It's probably safest to go the route you have taken. I withdraw the suggestion.